pkg/cli/cli.go

```diff
@@ -1358,7 +1358,13 @@ func runConnectionWebSocketAgentCommand(flags *ConnectFlags, c *command, args []
 		},
 	}
 	device := args[0]
-	wsUrl := fmt.Sprintf("%s/devices/%s/adb", wsRoot, device)
+	var wsUrlBuilder strings.Builder
+	wsUrlBuilder.WriteString(wsRoot)
```

Comment:
[docker-image-v0.1.0-alpha](https://google.github.io/styleguide/go/decisions.html#initialisms) In go, acronyms should have the same case, it should be `wsURLBuilder` in this case. However in the new function `wsURL()` you can have shorter names and avoid the `URL` part as it's going to be redundant.

Reply:
Changed to name to `url` to follow acronyms case rule and avoid confusion with the function name wsURL().